lib/amazon-pricing/definitions/ec2-instance-type.rb

```diff
@@ -143,6 +143,35 @@ def self.get_network_mbps(throughput)
       network_mbps
     end
 
+    # Take in string from amazon pricing api, return network properties
+    # Input: String containing network capacity from amazon-pricing-api
+    # Output: network throughput as a string, int containing network mbps
+    def self.get_network_information(network_string)
+      throughput = nil
+      case network_string
```

Comment:
can we convert all the case when's to a hash, just like @Network_Throughput_MBits_Per_Second?